sources/packages/backend/libs/services/src/queue/queue.module.ts

```diff
@@ -25,6 +30,12 @@ import { DatabaseModule } from "@sims/sims-db";
       inject: [ConfigService],
     }),
     BullModule.registerQueueAsync(...getQueueModules()),
+    BullBoardModule.forRootAsync({
```

Comment:
As discussed, the queue module is shared with API (to publish messages to queue) and loading bull board module here will stand up the bull dashboard in SIMS API as well.

Reply:
The modules are now added only to the queue-consumers.